t/rosie-disco/00-basic.t

```diff
@@ -46,30 +42,17 @@ __ROSE_CONF__
 export ROSE_CONF_PATH=$PWD
 $ROSE_HOME/sbin/rosa db-create -q
 #-------------------------------------------------------------------------------
-TEST_KEY=$TEST_KEY_BASE-rosie-disco
-rosie disco 'start' "${PORT}" \
-    0<'/dev/null' 1>'rosie-disco.out' 2>'rosie-disco.err' &
-ROSIE_DISCO_PID=$!
-T_INIT=$(date +%s)
-while ! port_is_busy $PORT && (($(date +%s) < T_INIT + 60)); do
-    sleep 1
-done
-if port_is_busy $PORT; then
-    pass "$TEST_KEY"
-else
-    fail "$TEST_KEY"
-    kill "${ROSIE_DISCO_PID}" 2>'/dev/null'
-    wait 2>'/dev/null'
-    rm -f ~/.metomi/rosie-disco-0.0.0.0-${PORT}* 2>'/dev/null'
+rose_ws_init 'rosie' 'disco'
+if [[ -z "${TEST_ROSE_WS_PORT}" ]]; then
```

Comment:
Should we have a `rose_ws_kill` here too or will the lack of setting that variable guarantee that the server never started?